dio/lib/src/options.dart

```diff
@@ -40,6 +42,27 @@ enum ResponseType {
   bytes
 }
 
+// CollectionFormat specifies the array format
+// (a single parameter with multiple parameter or multiple parameters with the same name)
+// and the separator for array items.
+enum CollectionFormat {
+  // Comma-separated values
+  // e.g. (foo,bar,baz)
+  csv,
+  // Space-separated values
+  // e.g. (foo bar baz)
+  ssv,
+  // Space-separated values
+  // e.g. (foo\tbar\tbaz)
+  tsv,
```

Comment:
are these comments correct?

Reply:
No, it's my mistake. It's should be "Tab-separated values"

Reply:
It's fixed